src/main/java/nextstep/step2/domain/Line.java

```diff
@@ -0,0 +1,60 @@
+package nextstep.step2.domain;
+
+import nextstep.step2.generator.BooleanGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Line {
+
+    static final String LADDER = "|";
+    static final String LINE_LADDER = "-----";
+    static final String BLANK = "     ";
+
+    private List<Boolean> points = new ArrayList<>();
+
+    public Line(int countOfParticipant, BooleanGenerator generator) {
+        IntStream.range(0, countOfParticipant - 1)
+                .forEach(i -> this.add(i, generator.generator()));
+    }
+
+    public Line(Boolean... booleans) {
+        IntStream.range(0, booleans.length)
+                .forEach(i -> this.add(i, booleans[i]));
+    }
+
+    private void add(int i, boolean generator) {
+        points.add(existPoint(i) ? false : generator);
```

Comment:
삼항연산자 또한 if-else 패턴과 같고 가독성 문제로 지양하는 편입니다.
다음과 같이 변경해 볼 수 있을 것 같습니다.
```suggestion
        if (!existPoint(i)) {
                        return;
        }
        points.add(generator);
```